systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceOperation.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources;
+
+import io.strimzi.api.kafka.model.Kafka;
+import io.strimzi.api.kafka.model.KafkaBridge;
+import io.strimzi.api.kafka.model.KafkaConnect;
+import io.strimzi.api.kafka.model.KafkaConnectS2I;
+import io.strimzi.api.kafka.model.KafkaConnector;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker2;
+import io.strimzi.api.kafka.model.KafkaRebalance;
+import io.strimzi.systemtest.Constants;
+
+import java.time.Duration;
+
+public class ResourceOperation {
+    public static long getTimeoutForResourceReadiness(String kind) {
+        long timeout;
+
+        switch (kind) {
+            case Kafka.RESOURCE_KIND:
+                timeout = Duration.ofMinutes(14).toMillis();
+                break;
+            case KafkaConnect.RESOURCE_KIND:
+            case KafkaConnectS2I.RESOURCE_KIND:
+            case KafkaMirrorMaker2.RESOURCE_KIND:
+            case Constants.DEPLOYMENT_CONFIG:
+                timeout = Duration.ofMinutes(10).toMillis();
+                break;
+            case KafkaRebalance.RESOURCE_KIND:
```

Comment:
Rebalance and Connector readiness will take much less time than others

Reply:
Yeah, good point, gonna move it. I wasn't 100% sure where to put it